### ISSUE TITLE
Cannot grab comments

### ISSUE THREAD

**Issue by packoverflowruss** [Russell-Newton/TikTokPy#9]
`UserWarning: Was unable to collect comments.
A second attempt might work.`

```
from tiktokapipy.api import TikTokAPI

video_url = "https://www.tiktok.com/@lindamaittm/video/7183294139855850794"

with TikTokAPI(navigation_retries=2, navigation_timeout=10, scroll_down_time=60) as api:
    video = api.video(video_url)
    print(video.comments)
```

**Comment by Russell-Newton** [comment]
Looking more into this issue, it looks like for whatever reason, when a Playwright chromium browser opens up TIkTok, only the request to collect comments from TikTok's API results in a CORS error. I'll keep looking into this.

**Comment by packoverflowruss** [comment]
> Looking more into this issue, it looks like for whatever reason, when a Playwright chromium browser opens up TIkTok, only the request to collect comments from TikTok's API results in a CORS error. I'll keep looking into this.

Thank you! Appreciate it.

**Comment by terrok9** [comment]
Hello!
I am having the same issue. Can I have an update of the current workaround? 
Maybe I can help.
Have a nice day.

**Comment by Russell-Newton** [comment]
Good idea. The place to update would be in `__enter__` in the sync API and `__aenter__` in the async API.

**Comment by terrok9** [comment]
I added playwright firefox support in order to solve this issue just for now #17.

**Comment by Russell-Newton** [comment]
Fixed in v0.1.9
